master/internal/trials/api_trials.go

```diff
@@ -55,16 +56,27 @@ func checkTrialFiltersEmpty(f *apiv1.TrialFilters) error {
 func (a *TrialsAPIServer) QueryTrials(ctx context.Context,
 	req *apiv1.QueryTrialsRequest,
 ) (*apiv1.QueryTrialsResponse, error) {
-	err := checkTrialFiltersEmpty(req.Filters)
+	curUser, _, err := grpcutil.GetUser(ctx)
+
+	if err != nil {
```

Comment:
Nit: stylistically, the error check is typically put up against the setting of the error without a blank line to group stuff into "paragraphs."

```go
foo, err := doSomething()
if err != nil {
	handle(err)
}

bar, err := doSomethingElse()
if err != nil {
	handle(err)
}
```

Reply:
ok, i have done this throughout